bundles/framework/personaldata/PublishedMapsTab.js

```diff
@@ -407,8 +471,16 @@ Oskari.clazz.define('Oskari.mapframework.bundle.personaldata.PublishedMapsTab',
             //sending a request to publisher for editing view
             var editRenderer = function (name, data) {
                 var link = me.templateLink.clone();
+                var srs = data.state.mapfull.state.srs;
                 link.text(name);
                 link.bind('click', function () {
+
+                    var supported = me._isCurrentProjectionSupportedForView( data );
+                    if (!supported) {
+                        me.createProjectionChangeDialog( me._loadProjectionUuid.bind( this, me.constructUrlUuid(srs) ) );
```

Comment:
This could be written as:

    me.createProjectionChangeDialog( function() {
        window.location.href = me.constructUrlUuid(srs);
     });

Which again would be easier to understand and you wouldn't need to browse the file to find the single-line implementation of _loadProjectionUuid()